fe1-web/src/core/network/__tests__/JsonRpcApi.test.ts

```diff
@@ -36,34 +35,36 @@ import {
   publish,
   subscribe,
 } from '../JsonRpcApi';
-import { getNetworkManager } from '../NetworkManager';
+import { SendingStrategy } from '../strategies/ClientMultipleServerStrategy';
 
 // region mock data
-const mockMessageData: MessageData = { object: ObjectType.ELECTION, action: ActionType.OPEN };
-
-const networkManager = getNetworkManager();
 
 // requires the mock registries to be set up
 // is initialized in beforeAll()
 let mockResponseMessage: Message;
 let mockResponse: JsonRpcResponse;
 
-const mockSendingStrategy = jest
+const mockMessageData: MessageData = { object: ObjectType.ELECTION, action: ActionType.OPEN };
+const mockSendingStrategy: SendingStrategy = jest
   .fn()
   .mockImplementation(() =>
     Promise.resolve([new ExtendedJsonRpcResponse({ receivedFrom: mockAddress }, mockResponse)]),
   );
 
+jest.mock('core/network/NetworkManager', () => {
+  const actual = jest.requireActual('/core/network/NetworkManager');
+  return {
+    ...actual,
+    getNetworkManager: () => new actual.TEST_ONLY_EXPORTS.NetworkManager(mockSendingStrategy),
```

Comment:
okay, that's one (effective) way to go about it that I didn't expect, I thought you'd just be mocking the one function 😅 
What's `TEST_ONLY_EXPORTS` ? I've never seen this 😅